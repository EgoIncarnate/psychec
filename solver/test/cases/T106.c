void f() {
  char x[A];
  char y[A][B];
  char z[A][B][C];
}
