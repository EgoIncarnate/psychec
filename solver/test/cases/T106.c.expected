int A;
int B;
int C;
